### ISSUE TITLE
Node cannot get locked object if it tried to delete one

### ISSUE THREAD

**Issue by ikovalev1** [nspcc-dev/neofs-node#1973]
## Expected Behavior
Any node must be able to get locked object  

## Current Behavior
Node cannot get locked object of other node if it tried to delete one

## Steps to Reproduce (for bugs)
1. on Node1 create  container with REP 1 e.g:
```
neofs-cli container create -r node1:8080 -w /etc/neofs/storage/wallet.json --policy "REP 1 IN n1 SELECT 1 FROM n1 AS n1 FILTER Node EQ node1 AS n1" --await --basic-acl public-read-write
```
2. put an object: 
```
neofs-cli object put -r node1:8080 -w /etc/neofs/storage/wallet.json  --file /home/service/.profile --no-progress --cid 
```
3.  lock the object:
```
neofs-cli object lock -r node1:8080 -w /etc/neofs/storage/wallet.json --expire-at 30 cid oid 
```
4. On other node (e.g. node2) try to delete the locked object
```
neofs-cli object delete  -r node2:8080 -w /etc/neofs/storage/wallet.json  --cid  HWcNK8ETNz263VerxrrNNQbwpPKAsGWKvgJZtePY6RVo --oid 9V3wLxh8eBgJAeAD61myJUf6oFm15e9LkSx9R7QhKFZ7
Enter password >
rpc error: remove object via client: status: code = 1024 message = incomplete object PUT by placement: could not close object stream: (*putsvc.remoteTarget) could not put object to [/dns4/node1.neofs/tcp/8080]: write object via client: status: code = 2050 message = object is locked
```
5. on the node of prev step (node2) try to get the object
```
neofs-cli object get -r node2:8080 -w /etc/neofs/storage/wallet.json --cid HWcNK8ETNz263VerxrrNNQbwpPKAsGWKvgJZtePY6RVo  --oid 9V3wLxh8eBgJAeAD61myJUf6oFm15e9LkSx9R7QhKFZ7
Enter password >
rpc error: read object header: status: code = 2052 message = object already removed
```
6. try to get this object from node1,3,4 - Ok
```
neofs-cli object get -r node1:8080 -w /etc/neofs/storage/wallet.json --cid HWcNK8ETNz263VerxrrNNQbwpPKAsGWKvgJZtePY6RVo  --oid 9V3wLxh8eBgJAeAD61myJUf6oFm15e9LkSx9R7QhKFZ7
Enter password >
< object content>
```
## Your Environment
root@az:/home/service# neofs-node --version
NeoFS Storage node
Version: v0.33.0-33-g60e9de8d
GoVersion: go1.18.4

**Comment by fyrchik** [comment]
Let's ensure that LOCK objects are broadcasted.

**Comment by carpawell** [comment]
Closed via #2008.
